.github/workflows/code-health.yml

```
@@ -65,6 +65,7 @@ jobs:
         uses: actions/setup-go@v4
         with:
           go-version-file: 'go.mod'
+          cache: false
```

can you add a comment linking to https://github.com/golangci/golangci-lint-action/issues/807
```suggestion
          cache: false # see https://github.com/golangci/golangci-lint-action/issues/807
```